native_client/BUILD

```diff
@@ -2,18 +2,27 @@
 
 cc_library(
     name = "deepspeech",
-    srcs = ["deepspeech.cc",
-            "c_speech_features/c_speech_features.c",
-            "kiss_fft130/kiss_fft.c",
+    srcs = ["deepspeech.cc"],
+    hdrs = ["deepspeech.h"],
+    deps = ["//tensorflow/core:tensorflow",
+            ":c_speech_features"]
+)
+
+cc_library(
+    name = "kissfft",
+    srcs = ["kiss_fft130/kiss_fft.c",
             "kiss_fft130/tools/kiss_fftr.c"],
-    hdrs = ["deepspeech.h",
-            "c_speech_features/c_speech_features.h",
-            "kiss_fft130/kiss_fft.h",
+    hdrs = ["kiss_fft130/kiss_fft.h",
             "kiss_fft130/_kiss_fft_guts.h",
             "kiss_fft130/tools/kiss_fftr.h"],
-    includes = ["c_speech_features",
-                "kiss_fft130"],
-    deps = [
-        "//tensorflow/core:tensorflow"
-    ]
+    includes = ["kiss_fft130"]
+)
+
+cc_library(
+    name="c_speech_features",
+    srcs = ["c_speech_features/c_speech_features.c"],
+    hdrs = ["c_speech_features/c_speech_features.h"],
+    includes = ["c_speech_features"],
+    deps = [":kissfft"],
+    copts = ["-std=gnu99"]
```

Comment:
Maybe just comment on why. At first glance, I tried using C99, and found out that M_PI is not defined there by default, hence GNU99:
`# We need C99, and it does not defines M_PI, so we need GNU99`